feat(dashboards): Add docs for Dashboard Edit Access Selector (#11822)
getsentry/sentry-docs: docs/product/dashboards/custom-dashboards/index.mdx

Dashboard filters are applied to all the widgets in your dashboard. They consist of [page filters](/concepts/search/#page-filters) that allow you to narrow down the widget results to certain projects, environments, and date ranges, and a release filter that restricts the data to certain releases. Changing a dashboard filter will prompt you to "Save" or "Cancel". To persist filter changes to your dashboard, click "Save", or click "Cancel" to discard the changes.

### Dashboard Edit Access

<Include name="feature-available-for-user-group-early-adopter.mdx" />

> 
> You can prevent other users from editing or deleting your dashboard by toggling the Edit Access Selector. The creator of the dashboard will always have edit access for their dashboards. To limit access to specific teams, use the Edit Access Selector to select teams. Clicking on "Save Changes" will update the Edit permissions for the dashboard. Only the dashboard's creator can modify edit permissions.
> 

Reviewer: ```suggestion

To restrict who can edit or delete your dashboard, go to the "Edit Access" selector. As the dashboard creator, you can limit access to specific teams by selecting the checkboxes in the Edit Access Selector and clicking Save Changes to apply the updates. As the dashboard creator, you always have edit access. 

```

## Widget Library

> The widget library contains a collection of prebuilt widgets you can add to your [custom dashboards](/product/dashboards/custom-dashboards/). Learn more in the [Widget Library documentation](/product/dashboards/widget-library/).

Reviewer: ```suggestion
The widget library contains a collection of prebuilt widgets you can add to your [custom dashboards](/product/dashboards/custom-dashboards/). Learn more by reading our [Widget Library docs](/product/dashboards/widget-library/).
```
